### ISSUE TITLE
Do not show the Flag menu if the current user has already flagged a review

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-frontend#5679]
If a user has already flagged a review, do not show them the "Flag" menu.

Note that the description of this issue has been changed from the original one reported by @ioanarusiczki as requested at https://github.com/mozilla/addons-frontend/issues/5679#issuecomment-418855792.

**Comment by bobsilverberg** [comment]
Is this something that works on the legacy site? @jvillalobos / @pwalm is this something we want to / need to support?

**Comment by jvillalobos** [comment]
I don't think unflagging is a common enough flow to worry about. I don't think it's useful to show the dropdown and allow more flagging if the review has already been flagged by that user, though.

**Comment by pwalm** [comment]
I wouldn't worry about unflagging either.

**Comment by bobsilverberg** [comment]
@jvillalobos So should we open a new issue about not showing the drop-down menu to allow more flagging if the user has flagged a review already, or should we just close this and leave it as it is?

**Comment by jvillalobos** [comment]
Yes, I think there should be a separate issue to remove the dropdown, and this one can be closed.

**Comment by bobsilverberg** [comment]
Rather than closing this and opening another, let's just re-purpose this issue.

This is now about not showing the "Flag" menu if the review has already been flagged by the user.

**Comment by kumar303** [comment]
Can you update this issue description or file a new issue? Otherwise it will be hard to dig through the comments and see what the actual feature request is.

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDFRNT-141
